CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ Unreleased
 - Alerting: Manage alerts for duplicate scheduled jobs (#5871)
 - Identity Verification: Add more phone number validation to phone confirmation (#5873)
 - Dependencies: Update various dependencies (#5877)
-- Maintenance: Enable stricter Changelog checks in continuous integration (#5882)
+- Maintenance: Enable stricter Changelog checks in continuous integration ([#5882](https://github.com/18F/identity-idp/pull/5882))
```

Comment:
maybe it's okay if these aren't markdown links directly in the `.md`? They get linkified correctly when copied to the actual release notes?

Reply:
It's kind of a bummer to not have the links work in the file itself. It's not critical and I'm ok leaving it out. Does it help if I added a flag to autocorrect and add the links to the changelog script in cb5b9ec41? 🙂

Would be nice if we could specify files to opt-in for auto-linking in GitHub.

Reply:
Maybe if there's a way we can do that with a simpler/shorter regex? I guess that's my motivation for commenting, is that the regex to support this seems like a little much, Maybe we could turn it into a more forgiving capture group and do the "is it a pull request" check with plain Ruby code?

I'm personally happy enough to rely on the linkification in Release notes only, but this is not the hill I die on

Reply:
It's hard to cover the common cases well without having the large regex unfortunately, and I don't think it's worthwhile if without autocorrect available. I'm not committed to it, so I've backed out those changes.